lib/internal/webstreams/readablestream.js

```diff
@@ -288,6 +289,12 @@ class ReadableStream {
     return isReadableStreamLocked(this);
   }
 
+  get streamClosed() {
+    if (!isReadableStream(this))
+      throw new ERR_INVALID_THIS('ReadableStream');
+    return this[kState].streamClosed.promise;
+  }
+
```

Comment:
You can't add new public properties to web streams. That is not allowed by the spec.

Reply:
ok updating this to private access?

Reply:
Have updated to access using [kState]